helm/alfresco-dbp/templates/NOTES.txt

```diff
@@ -6,14 +7,12 @@ The chart path can either be local "." on in the helm repo "alfresco/alfresco-db
 
 If you are using the default dbp setup you can access all components as follows:
 
-{{- $protocol := index (split ":" .Values.global.keycloak.url) "_0" -}}
-
-
   Alfresco Digital Workspace: {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/workspace/
   Content: {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/alfresco
   Share: {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/share
   Solr: {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/solr
   Alfresco Identity Service: {{ index .Values "global" "keycloak" "url" }}
-  APS: {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/activiti-app	
-  APS Admin: {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/activiti-admin
-  Sync Service {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/syncservice/healthcheck
\ No newline at end of file
+  APS: {{ $protocol }}://{{ index .Values "alfresco-process-services" "ingress" "hostName" }}/activiti-app
+  APS Admin: {{ $protocol }}://{{ index .Values "alfresco-process-services" "ingress" "hostName" }}/activiti-admin
+  Sync Service: {{ $protocol }}://{{ index .Values "alfresco-content-services" "externalHost" }}/syncservice/healthcheck
+  Process Workspace: {{ $protocol }}://{{ index .Values "alfresco-process-services" "ingress" "hostName" }}/process-workspace
```

Comment:
Could we move this to be before APS, similar to the precedence above where the default UI of content is first (ADW) then the older UIs and services follow.

Reply:
Done.